.gitignore

```diff
@@ -13,3 +13,6 @@
 vignettes/*
 !vignettes/*.Rmd
 .Rproj.user
+
+
+docs/*.
```

Comment:
Please also delete the entire `docs/` directory.  The Travis deployment will push the docs to the `gh-pages` branch instead of `master`.